src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -68,6 +68,20 @@ export namespace ValidationSchemas {
                     memoryMB: joi.number().min(100),
                     gpuNum: joi.number().min(0).required(),
                     command: joi.string().min(1).required()
+                }),
+                taskRoles: joi.array({
+                    name: joi.string().min(1),
+                    replicas: joi.number().min(1).required(),
```

Comment:
I remember frameworkcontroller uses taskNum instead of replicas. Should we consider to keep consistent with original frameworkcontroller's definition?

Reply:
updated to taskNum